xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminSheet.xml

```diff
@@ -1064,9 +1064,17 @@ document.observe('xwiki:dom:loaded', function() {
     width: 65%;
 }
 
+dl.rights-settings {
```

Comment:
not sure why you need to increase specificity by explicitly saying 'dl' - it could be removed.